docs/podman.1.md

```diff
@@ -39,6 +39,11 @@ Path to where the cpu performance results should be written
 
 log messages above specified level: debug, info, warn, error (default), fatal or panic
 
+**--namespace**
+
+set libpod namespace. Namespaces are used to separate groups of containers and pods in libpod's state.
```

Comment:
This should probably be capitalized (the “log” above is the only non-capitalized item).